compiler_opt/tools/generate_default_trace.py

```diff
@@ -206,6 +214,8 @@ def main(_):
                                     total_failed_examples, total_work)
 
         results = results_queue.get()
+        if isinstance(results, BaseException):
+          logging.fatal(results)
```

Comment:
`logging.fatal` doesn't insta-exit in the test environment, it errors out a couple lines down when trying to unpack the exception instead. Could switch to `logging.error` and reraise the exception

Reply:
probably just raise the error

Reply:
sgtm. left a comment to make sure we don't start catching that